cli/query/query.go

```diff
@@ -0,0 +1,99 @@
+package query
+
+import (
+	"bytes"
+	"fmt"
+	"strconv"
+	"text/tabwriter"
+
+	"github.com/nspcc-dev/neo-go/cli/options"
+	"github.com/nspcc-dev/neo-go/pkg/core/transaction"
+	"github.com/nspcc-dev/neo-go/pkg/encoding/fixedn"
+	"github.com/nspcc-dev/neo-go/pkg/rpc/response/result"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	"github.com/urfave/cli"
+)
+
+// NewCommands returns 'wallet' command.
+func NewCommands() []cli.Command {
+	queryTxFlags := append([]cli.Flag{
+		cli.BoolFlag{
+			Name:  "verbose, v",
+			Usage: "Output full tx info and execution logs",
+		},
+	}, options.RPC...)
+	return []cli.Command{{
+		Name:  "query",
+		Usage: "query",
+		Subcommands: []cli.Command{
+			{
+				Name:   "tx",
+				Usage:  "query tx status",
+				Action: queryTx,
+				Flags:  queryTxFlags,
+			},
+		},
+	}}
+}
+
+func queryTx(ctx *cli.Context) error {
+	args := ctx.Args()
+	if len(args) == 0 {
+		return cli.NewExitError("Transaction hash is missing", 1)
+	}
+
+	txHash, err := util.Uint256DecodeStringLE(args[0])
+	if err != nil {
+		return cli.NewExitError(fmt.Sprintf("Invalid tx hash: %s", args[0]), 1)
+	}
+
+	gctx, cancel := options.GetTimeoutContext(ctx)
+	defer cancel()
+
+	c, err := options.GetRPCClient(gctx, ctx)
+	if err != nil {
+		return cli.NewExitError(err, 1)
+	}
+
+	tx, err := c.GetRawTransaction(txHash)
+	if err != nil {
+		return cli.NewExitError(err, 1)
+	}
+
+	res, _ := c.GetApplicationLog(txHash, nil)
```

Comment:
Let's handle this error. Although we're sure that tx exists, the server still may not support application logs (missing C# ApplicationLogs module) or something else can happen with the connection.

Reply:
Fixed. Filtering by `Unknown transaction` seems to work for both C# and neo-go implementations.